jupyter_platform/cubeviz_leaflet_prototype_1.ipynb

```diff
@@ -231,8 +216,8 @@
    "source": [
     "url_templ = 'http://127.0.0.1:5013/fitsNUM/{z}/{x}/{y}.png'\n",
     "\n",
-    "m = ipyleaflet.Map(center=(0, 0), zoom=1, layers=[], \n",
-    "                   min_zoom=1, scroll_wheel_zoom=True)\n",
+    "m = ipyleaflet.Map(crs='Simple', center=(0, 0), zoom=1, min_zoom=1,\n",
```

Comment:
@eteq , this does not look as Simple as it sounds. I need to stare at https://leafletjs.com/examples/crs-simple/crs-simple.html a bit more...